public/components/custom_panels/home.tsx

```diff
@@ -297,9 +299,10 @@ export const Home = ({
         side={toastRightSide ? 'right' : 'left'}
         toastLifeTimeMs={6000}
       />
+      <Switch>
```

Comment:
Seems like indent issue, did you bootstrap your app and see husky running when you committing your code?

Reply:
Will check now